AcademyApp/Designables/UIButtonX.swift

```diff
@@ -0,0 +1,29 @@
+//
+//  UIButtonX.swift
+//  AcademyApp
+//
+//  Created by Peter Sevcik on 14/06/2019.
+//  Copyright © 2019 Peter Sevcik. All rights reserved.
+//
+
+import UIKit
+
+@IBDesignable
+class UIButtonX: UIButton {
```

Comment:
Do you expect future inheritance of this class? If not its good habit to make those kind of class as final...
More about reducing dynamics here
https://developer.apple.com/swift/blog/?id=27

Reply:
Marked RoundedCornersButton class as final.